### PR TITLE
docs/ppdsc 2616 accessibility section

### PR DESCRIPTION
**Pull request by baburay23**
PPDSC-XXXX

**What**

1. Background - why this is needed
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2616-accessibility-section
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2616-accessibility-section/storybook

File: site/pages/getting-started/accessibility/write.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+              How to write in plain English
+            </Link>{' '}
+            - from the Plain English Campaign.
+          </P>
+          <br />
+
+          <P overrides={{typographyPreset: 'editorialParagraph030'}}>
+            <Link
+              href="https://sites.google.com/news.co.uk/d-and-i/inclusive-terminology?authuser=0"
+              overrides={{typographyPreset: 'editorialParagraph030'}}
```
Comment (mutebg) [inline site/pages/getting-started/accessibility/write.tsx:146]: I belive if you pass `type="inline" you will not need to add typographyPreset ( that apply to all links which are in paragraph )

File: site/pages/getting-started/accessibility/testing.tsx [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+      <ContentSection sectionName="test accessible websites">
+        <ContentPrimary
+          id="test-design"
+          headline="Test your design"
+          toc="Test your design"
+          description={
+            <>
+              <br />
+              <br />
+              <UnorderedList
```
Comment (mutebg) [inline site/pages/getting-started/accessibility/testing.tsx:68]: that might be children of ContentPrimary instead of description. Most of the time this kind of "rich" content is like children instead of description

**Review by JohnTParsons — APPROVED** [review]

File: site/pages/getting-started/accessibility/testing.tsx [hunk 3]
```diff
@@ -61,155 +61,82 @@ const AccessibilityTest = (layoutProps: LayoutProps) => (
...
+                },
+                marker: {
+                  spaceInline: 'space020',
+                },
+              }}
+            >
+              <>
+                <Link
+                  href="https://webaim.org/resources/contrastchecker/"
+                  overrides={{typographyPreset: 'editorialParagraph030'}}
```
Comment (mutebg) [inline site/pages/getting-started/accessibility/testing.tsx:83]: Does it work to use type="inline" here so that inherits the typography preset from its parent?

**Review by mstuartf — APPROVED** [review]

**Review by mutebg — APPROVED** [review]

**Review by JohnTParsons — APPROVED** [review]
